supabase/migrations/20220402234420_initial.sql

```diff
@@ -0,0 +1,168 @@
+-- This script was generated by the Schema Diff utility in pgAdmin 4
```

Comment:
Is there any chance this could have been done with the by hand SQL? That way it keeps all the comments?

Reply:
I have resolved this so I can continue to edit the tables in another PR